sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientClientLiveTests.cs

```diff
@@ -754,7 +754,7 @@ private record TestModelForTypesNullable
             public BinaryData Dynamic { get; set; }
         }
 
-        [Test]
+        [RecordedTest]
```

Comment:
Shouldn't this result in a .json recording file being added in the PR too?  Or am I so out of touch I no longer understand all the machinery in our test framework?  😄

Reply:
The test recordings were added in the last pr, I think this header got mixed up and it got merged incorrectly as [Test]